### PR TITLE
update macOS installation instructions

### PR DESCRIPTION
**Pull request by fakerybakery**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,18 +95,29 @@ Latest installers with Mamba and PyPy in the base environment:
...
 or
 
-    wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-    bash Miniforge3-$(uname)-$(uname -m).sh
+```bash
+wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh
+```
+
+**NOTE:** For macOS, you may need to run `bash` with `sudo`:
```
Comment (hmaarrfk) [inline README.md:115]: I'm not sure this is correct.

We actually prefer that users not use sudo unless they really know what they are doing, then they don't need us to tell them to use sudo. The preferred way is simply to install in your user's home directory like the default will do.

Can you expand on the usecase where you find `sudo` necessary or desired?

Comment (hmaarrfk) [reply README.md:115]: it should be attempting to change your personal zshrc which you should have access to.

That is, it should be trying to edit (on OSX)
```
/Users/fakerybakery/.zshrc
```

which should be a file you control, that is writable by your user.

It maybe that you are running into a real bug.

If you can recreate the traceback and the error that would be really useful.

I don't have an OSX machine, but others might be able to help troubleshoot.

Comment (fakerybakery) [reply README.md:115]: I got:
```
TypeError: memoryview: a bytes-like object is required, not 'str'
```

Comment (jakirkham) [reply README.md:115]: This is likely a bug in the installer code. If you can share more of the traceback (or better yet how the install was done), that would help in identifying where this comes from

**Pull request closed without merging**